location/models.py

```diff
@@ -67,4 +68,4 @@ def __str__(self):
 
     @property
     def estimated_rent_price_for_display(self):
-        return f"${self.estimated_rent_price}"
+        return f"${self.rent_price}"
```

Comment:
should this be changed if the method is `estimated_rent_price...`? Or should we update the method signature to reflect what it is returning or add a different method to return the actual rent price?

Reply:
Good point! Completely forgot to change the method name. Regarding having 2 different methods, I discussed this earlier with Yonguk and it seemed like the default rent price is stored in the `rent_price` attribute. In either case, if we are getting data from landlord or from zillow, this field should contain the actual rental price that would be displayed in the location view.